module/Api/src/Service/SecretsManager/LocalSecretsManager.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Dvsa\Olcs\Api\Service\SecretsManager;
+
+class LocalSecretsManager extends AbstractSecretsManager implements SecretsManagerInterface
+{
+    private array $config;
+
+    public function __construct(array $config)
+    {
+        $this->config = $config;
+    }
+
+    public function getSecret(string $secretName): array
```

Comment:
As mentioned in the last couple of stand ups, a ticket is already created for this:
https://dvsa.atlassian.net/browse/VOL-5650

Reply:
Yeah I hear ya - it's because the branch is absolutely ancient, it'll have pre dated all of that. I'm assuming that's why it was done that way, but Shaun could confirm.